### PR TITLE
Added `water contrib` command! 

### LINKED ISSUE #37: Add a `water contribute` command
We're an Open Source repository and what are we without contributions!
Add a command to let people know where they can go to contribute

### PR DESCRIPTION
**Pull request by mukunddeepak**
#37: Added the contribute command.
P.S. : Let me know if you need me to make any changes and make another pull request. 

Edit:
Closes #37

### CODE REVIEW

File: commands/contribute.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            .setColor("#0099ff")
+            .setTitle("DrinkWaterBot Contribute")
+            .setDescription("Use `water contrib` to see how you can contribute!")
+        msg.channel.send(embed)
+    }
+}
+
+
+module.exports = {
+    contripy: contripy
```
Comment (srujandeshpande) [inline commands/contribute.js:32]: Is there any particular reason for the function "contripy"? If not, try changing it to something more descriptive of the command

**Review by srujandeshpande — CHANGES_REQUESTED** [review]

**Comment by srujandeshpande** [comment]
Also, 'water help contribute` isn't working that's why I haven't merged it yet. Any idea why?

File: commands/contribute.js (hunk 1 above)
Comment (mukunddeepak) [reply commands/contribute.js:32]: ok sure ill do that

**Comment by mukunddeepak** [comment]
Shoot! I forgot to add that. on it!

File: commands/contribute.js (hunk 1 above)
Comment (mukunddeepak) [reply commands/contribute.js]: New pull request on the way!

Comment (mukunddeepak) [reply commands/contribute.js]: I have added new commit to same pull request with requested changes!

**Comment by mukunddeepak** [comment]
I have added new commit to same pull request with requested changes!

**Review by srujandeshpande — COMMENTED** [review]
Works great, thanks!

**Pull request merged**